app/code/core/Mage/Payment/Model/Method/Cashondelivery.php

```diff
@@ -52,4 +52,19 @@ public function getInstructions()
     {
         return trim($this->getConfigData('instructions'));
     }
+
+    /**
+     * Not available for quote without delivery
+     *
+     * @param Mage_Sales_Model_Quote $quote
+     * @param int|null $checksBitMask
+     * @return bool
```

Comment:
I'm not 100% sure but maybe we can use:
```suggestion
     * @inheritdocl
```